content/docs/dvclive/how-it-works.md

```diff
@@ -71,20 +71,62 @@ model.pt.dvc
 ## Track the results
 
 DVCLive expects each run to be tracked by Git, so it will save each run to the
-same path and overwrite the results each time. If you
-[run experiments](/doc/user-guide/experiment-management/running-experiments)
-using DVC <abbr>pipelines</abbr>, DVC will track them for you so you don't have
-to Git commit each run. <abbr>DVC experiments</abbr> are Git commits that DVC
-can find but that don't clutter your Git history or create extra branches in
-your repo.
+same path and overwrite the results each time. Include
+[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to auto-track
+as a <abbr>DVC experiment</abbr>. DVC experiments are Git commits that DVC can
+find but that don't clutter your Git history or create extra branches.
 
-If you don't have a DVC pipeline, you can include
-[`save_dvc_exp=True`](/doc/dvclive/api-reference/live#parameters) to save the
-results as a DVC experiment.
+### Track large artifacts
 
-When using `Live.log_artifact("model.pt")`, DVCLive will
+Models and data are often large and aren't easily tracked in Git.
+`Live.log_artifact("model.pt")` will
 [cache](/doc/start/data-management/data-versioning) the `model.pt` file with DVC
-to avoid tracking large artifacts in Git. It will generate a `model.pt.dvc`
-metadata file, which you should track in Git. You can
-[retrieve](/doc/start/data-management/data-versioning#retrieving) the artifact
-from the Git commit.
+and make Git ignore it. It will generate a `model.pt.dvc` metadata file, which
+can be tracked in Git and becomes part of the experiment. With this metadata
+file, you can [retrieve](/doc/start/data-management/data-versioning#retrieving)
+the versioned artifact from the Git commit.
+
+### Run experiments with DVC pipelines
+
+You may
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
+using DVC <abbr>pipelines</abbr>. When your experiment completes, DVCLive prints
+instructions for how to start:
+
+```
+Put this into `/Users/dave/Code/myproject/dvc.yaml` to run experiments with DVC:
+
+stages:
+  dvclive:
+    cmd: # Insert command to run the code
+    deps:
+      - train.py
+    outs:
+      - dvclive/metrics.json:
+          cache: false
+      - dvclive/plots:
+          cache: false
```

Comment:
I though we agree that we don't really need this. 
Is it because the omission would be harder to explain?

Reply:
I thought we agreed on it because it was easier than explaining this. I hadn't really considered leaving it out TBH. If we show it like this, what is the downside?

> Is it because the omission would be harder to explain?

I can think of a few reasons:
1. Yep, it's confusing to explain why we include some outputs and not others. 
2. Without it, the `outs` might often be empty, which might also be confusing and unhelpful.
3. It gives you a hint about what files have been generated.

Reply:
> If we show it like this, what is the downside?

The downside is that by including it we will disable the run cache now since we include `cache: false` outputs 🤔 . Feels more like a limitation of how we handle `cache: false` outputs. Goes back to discussion from https://github.com/iterative/dvc/issues/4428 that we may need different flags for whether to cache and whether to git track/ignore. For outputs like metrics, we probably want to still cache since it's cheap but also leave a copy in git to make it more accessible.